applications/staex-iod.md

```diff
@@ -0,0 +1,326 @@
+# StaexIoD
+
+- **Team Name:** Staex
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 3
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+Staex Internet of Data.
+
+The goal of this project is to create a Web3 IoT data infrastructure with a stable economy. In simple words we want to simplify Web3 onboarding process and bring more IoT device owners to share their useful data with some profit and other people to find and use such data transparently, securely and easily for their life or research.
+
+We want to bring Web3 data access infrastructure to your ecosystem. As a part of Web3 Foundation ecosystem we want to use Kusama for our smart contracts.
+
+Our team is interested in creating this project because Staex are working on several solutions for IoT devices, so we are fully interesting to propose such topics to community and improve IoT field at all with new solutions.
+
+### Project Details
+
+#### Architecture
+
+![architecture](https://raw.githubusercontent.com/lavrd/cdn/main/image.png)
+
+#### IoT devices indexing
+
+To give users possibility to find particular type of data or particular IoT device we need to index them somehow. Indexing part we want to implement through DIDs and Subsquid project.
+
+So we will implement ink! smart contract which will be deployed on Kusama network. This smart contract will have several methods:
```

Comment:
Kusama isn't capable of running smart contracts since it is a relay chain. Did you mean to list a different chain or testnet here?

Reply:
Yes, it was a mistake from me. While this phase of the project, maybe, it is not necessary to deploy smart contracts on testnet and for sure on mainnet. But for future it seems that we need to deploy them to Shiden parachain or to Shibuya while testing.